pcfuncs/funclib/stamps/stamp.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+from typing import TYPE_CHECKING
+
+from PIL.Image import Image as PILImage
+
+if TYPE_CHECKING:
+    from animation.frame import AnimationFrame
+
+TRANSPARENT = (255, 255, 255, 0)
+
+
+class FrameStamp(ABC):
+    def __init__(self, frame: "AnimationFrame") -> None:
```

Comment:
FYI, when stamps get brought into the /image endpoint, the typing here will need to change. Animations need the per-frame state.